packages/@okta/vuepress-site/docs/release-notes/2019-06-3/index.md

```diff
@@ -17,4 +17,7 @@ The [Token Inline Hook](/docs/reference/token-hook/) now lets you modify particu
 
 * When a customer used a [Token Inline Hook](/docs/reference/token-hook/) and returned an `error` object to Okta, Okta failed to pass the error to the token requester. (OKTA-231397)
 
-* The issuer claim inside JWT tokens was erroneously changing to all lowercase causing JWT verification failure when the application was case-sensitive. (OKTA-235710)
\ No newline at end of file
+* The issuer claim inside JWT tokens was erroneously changing to all lowercase causing JWT verification failure when the application was case-sensitive. (OKTA-235710)
+
+* When a customer called the `POST /idps/credentials/keys` endpoint and supplied an `x5t#S256` parameter to specify the SHA-256 thumbprint of the certificate they were adding, Okta failed to validate the thumbprint.
```

Comment:
When a customer called the `POST /idps/credentials/keys` endpoint and supplied an `x5t#S256` parameter to specify the SHA-256 thumbprint of the certificate that they were adding, Okta failed to validate the thumbprint.

Reply:
Made the edit